app/lib/search/backend.dart

```diff
@@ -149,11 +150,11 @@ class SearchBackend {
 }
 
 class PopularityStorage {
-  final String _latestPath = 'popularity-latest.json.gz';
+  static final String _latestPath = 'popularity-latest.json.gz';
```

Comment:
How about const?

Reply:
There's a non-zero cost in the VM to try to canonicalize const values IIRC, so I generally avoid 'em unless there's a big win.

Not like this change is huge, either. Not many instances of `PopularityStorage` during a run...